pkg/machines/components/create-vm-dialog/createVmDialog.jsx

```diff
@@ -732,7 +734,7 @@ class CreateVmModal extends React.Component {
 
     render() {
         const { nodeMaxMemory, nodeDevices, networks, osInfoList, loggedUser, providerName, storagePools, vms } = this.props;
-        const validationFailed = this.state.validate && validateParams({ ...this.state, osInfoList });
+        const validationFailed = this.state.validate && validateParams({ ...this.state, osInfoList }, vms.filter(vm => vm.connectionName == this.state.connectionName));
```

Comment:
Why you don't keep the new paramers inside the dict to allow it be named parameter?
Then you can access it with vmParams.vms from the function.
It's just weird to have some args names and some non.

Reply:
Oh I tought that `vmParams` refers to just dialog state. Now I noticed it also contais osInfoList, so yeah, it would be fine to put it there.